CHANGELOG.md

```diff
@@ -11,6 +11,11 @@ Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how
 
 - `MIRRORD_AGENT_NETWORK_INTERFACE` environment variable/file config to let user control which network interface to use. Workaround for [#670](https://github.com/metalbear-co/mirrord/issues/670).
 
+### Changed
+
+- VS Code Extension: change extension to use a mirrord-config file for configuration
```

Comment:
I think this is major (breaking) no?

Reply:
Keeping this a minor since it only affects the VSCode extension